helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -177,6 +177,8 @@ starrocksFESpec:
     # - name: my-configmap
     #   mountPath: /etc/my-configmap
     #   subPath: ""
+  # terminationGracePeriodSeconds defines duration in seconds the FE pod needs to terminate gracefully.
```

Comment:
why there is only 3 occurrence of the `terminationGracePeriodSeconds` while there are 4 changes in the starrocks_cluster.yaml?

Reply:
The terminationGracePeriodSeconds was placed in an incorrect location, allowing the FE Proxy to also set the terminationGracePeriodSeconds.